l10n_ar_ux/models/res_config_settings.py

```diff
@@ -16,6 +16,13 @@ class ResConfigSettings(models.TransientModel):
         help="Si marca esta opción, cuando se imprima o envíe un Recibo de Clientes, se incluirá"
         " una sección con todos los Comprobantes abiertos, es decir, que tengan algún saldo pendiente",
     )
+    l10n_ar_afip_activity_id = fields.Many2one(
+        "afip.activity",
+        string="Principal Activity",
+        help="Principal registered activity of the company",
+        related="company_id.l10n_ar_afip_activity_id",
+        readonly=False,
+    )
```

Comment:
remove this lines, all the attributes of the original field are copied.
you will need to define only realated and readonly attribute

```suggestion
    l10n_ar_afip_activity_id = fields.Many2one(
        related="company_id.l10n_ar_afip_activity_id",
        readonly=False,
    )
```